packages/odyssey-react-mui/README.md

```diff
@@ -118,6 +118,14 @@ const YourAppRoot = ({ children }) => (
 );
 ```
 
+### Generating ok-PL and ok-SK files
+
+Right now we do not have the code to generate these files in Odyssey.
+
+If you are an Okta employee, the easiest way to generate these files is to replace the English properties file of another project and run the build commands to get the ok-PL and ok-SK versions. For ok-SK, you will also need to replace the prefix with `odyssey:odyssey-react-mui: `.
```

Comment:
not optimal but we have a JIRA to address this here: https://oktainc.atlassian.net/browse/OKTA-622636